docs/rule/no-attrs-in-components.md

```diff
@@ -0,0 +1,18 @@
+## no-attrs-in-components
+
+This rule prevents the usage of `attrs` property to access values passed to the component since all the values can be accessed directly from the from the template.
+
+For example, doing
+
+```components/templates/layout.hbs
+{{attr.foo}}
+```
+is bad. The correct way would be to directly consume `foo` in the template like,
```

Comment:
Can you also add a "good" example using `@foo` (works in Ember 3.1+)?

Reply:
Done